src/ui/app_view.rs

```diff
@@ -357,10 +357,9 @@ pub fn app_view(search_trigger: Trigger) -> impl View {
 		});
 
 	let main_window = scroll(
-		dyn_container(
-			move || active_tab.get(),
-			move |id| detail_view(id, main_scroll_to).any(),
-		)
+		dyn_container(move || {
+			untrack(|| detail_view(active_tab.get(), main_scroll_to).into_any())
```

Comment:
Should remove the untrack from here, it stops the main window from changing

Reply:
You are of course very right! The `untrack` makes me nervous for all those side effects further down the line. From the discussions in discord it sounds like there will be a `dyn_container` soon that has the old functionality without subscribing signals automatically. So maybe we wait until that work is merged in before merging this PR?